pypesto/hierarchical/parameter.py

```diff
@@ -10,6 +10,7 @@ class InnerParameter:
     SCALING = 'scaling'
     OFFSET = 'offset'
     SIGMA = 'sigma'
+    OPTIMALSCALING = 'optimalScaling'
```

Comment:
in what way is this different from the scaling? can one indicate that this is based on qualitative data -> `qualitative_scaling`?

Reply:
The method is called optimal scaling. But also fine for me to change the name here, as its just about the parameter

Reply:
but this "optimalScaling" is supposed to imply the qualitative aspect already, right? then probably better to include that in the name. if "scaling" can be used here (I guess not, because the method needs to know that it needs to apply the qualitative solver), also fine.

Reply:
Any keyword would work for the method. Makes sense to change it to something clearly indicating, that its about qualitative data :+1:

Reply:
Changed it to `qualitative_scaling`